cli/installer/install-azd.ps1

```diff
@@ -25,11 +25,35 @@ Do not update the PATH environment variable with the location of azd.
 
 .PARAMETER DownloadTimeoutSeconds
 Download timeout in seconds. Default is 120 (2 minutes).
+
+.EXAMPLE
+powershell -ex AllSigned -c "Invoke-RestMethod 'https://aka.ms/install-azd.ps1' | Invoke-Expression"
+
+Install the azd CLI from a Windows shell
+
+The use of `-ex AllSigned` is intended to handle the scenario where a machine's
+default execution policy is restricted such that modules used by
+`install-azd.ps1` cannot be loaded. Because this syntax is piping output from
+`Invoke-RestMethod` to `Invoke-Expression` there is no direct valication of the
+`install-azd.ps1` script's signature. Validation of the script can be
+accomplished by downloading the script to a file and executing the script file.
+
+.EXAMPLE
+Invoke-RestMethod 'https://aka.ms/install-azd.ps1' -OutFile 'install-azd.ps1'
+PS > ./install-azd.ps1
+
+Download the installer and execute from PowerShell
+
+.EXAMPLE
+Invoke-RestMethod 'https://aka.ms/install-azd.ps1' -OutFile 'install-azd.ps1'
+PS > ./install-azd.ps1 -Version daily
+
+Download the installer and install the "daily" build
 #>
 
 param(
-    [string] $BaseUrl = "https://azure-dev.azureedge.net/azd/standalone/release",
-    [string] $Version = "latest",
+    [string] $BaseUrl = "https://azuresdkreleasepreview.blob.core.windows.net/azd/standalone/release",
```

Comment:
Did you intend to change the base URL to not use the CDN and the Version to daily?

Reply:
Fixed. Fortunately this did not propagate to the shell scripts.